src/mmw/js/src/modeling/views.js

```diff
@@ -66,10 +67,83 @@ var ModelingHeaderView = Marionette.LayoutView.extend({
 // The drop down containing the project name
 // and projects options drop down.
 var ProjectMenuView = Marionette.ItemView.extend({
-    model: models.ProjectModel,
+    ui: {
+        rename: '#rename-project',
+        share: '#share-project',
+        remove: '#delete-project',
+        print: '#print-project'
```

Comment:
I've started to use `data-action="rename"` for these types of elements. Not sure if that's the best way to go, but just FYI.